src/main/java/net/minecraftforge/fmp/item/IItemSaw.java

```diff
@@ -0,0 +1,10 @@
+package net.minecraftforge.fmp.item;
+
+import net.minecraft.item.ItemStack;
+
+public interface IItemSaw
+{
```

Comment:
Docs, does this even need to be part of this PR? Or, should this bee moved to a capability...